modules/messages/src/lib.rs

```diff
@@ -1469,29 +1479,38 @@ mod tests {
 	fn storage_message_key_computed_properly() {
 		// If this test fails, then something has been changed in module storage that is breaking all
 		// previously crafted messages proofs.
+		let storage_key = storage_keys::message_key::<TestRuntime, DefaultInstance>(&*b"test", 42).0;
 		assert_eq!(
-			storage_keys::message_key::<TestRuntime, DefaultInstance>(&*b"test", 42).0,
-			hex!("87f1ffe31b52878f09495ca7482df1a48a395e6242c6813b196ca31ed0547ea79446af0e09063bd4a7874aef8a997cec746573742a00000000000000").to_vec(),
+			storage_key,
+			hex!("dd16c784ebd3390a9bc0357c7511ed018a395e6242c6813b196ca31ed0547ea79446af0e09063bd4a7874aef8a997cec746573742a00000000000000").to_vec(),
```

Comment:
Why did this change?

Reply:
Afaik the storage key is computed as `twox(pallet-name ++ storage-key-name)` => then pallet name has been changed